distribution/ltp-upstream/include/knownissue.sh

```diff
@@ -165,6 +165,8 @@ function knownissue_filter()
 	tskip "pty04" unfix
 	# https://lists.linux.it/pipermail/ltp/2020-May/017044.html
 	tskip "nm01_sh" unfix
+	# https://github.com/linux-test-project/ltp/issues/611
+	tskip "kms02" unfix
```

Comment:
ksm02 and ksm02_1

Reply:
@jstancek thanks I missed that, I can change to `ksm02.*`

Reply:
https://github.com/CKI-project/tests-beaker/pull/531

Reply:
This should be 'ksm02.*' but not 'kms02.*', looks like a typo.

Reply:
gah! fixed the typo too https://github.com/CKI-project/tests-beaker/pull/532/files